types/word-extractor/word-extractor-tests.ts

```diff
@@ -0,0 +1,13 @@
+import * as WordExtractor from "word-extractor";
```

Comment:
Generally speaking, `import *` creates a symbol that can't be instantiated with `new`.  Did you mean `import WordExtractor = require("word-extractor");`?

Reply:
Force-pushed a new revision with the fix.